.changeset/shaggy-fireants-try.md

```diff
@@ -0,0 +1,9 @@
+---
+'@braid-design-system/docs-ui': patch
+---
+
+Add Braid 'docs-ui' package to the Braid Docs Site for usage.
+
+As part of a larger effort to migrate the Docs Site components to the Braid 'docs-ui' package:
+* Migrate usage of the **LinkableHeading** component from Braid `src` to Braid `docs-ui` package.
+* Remove the no longer used **LinkableHeading** component from Braid `src`.
```

Comment:
Realising that this changeset is really the PR description. The changeset is really only capturing the removal of the space character. Something like:

```suggestion
**LinkableHeading:** Remove space character between heading and link icon, instead use only margin value.
```